src/main/java/io/swagger/template/TemplateApi.java

```diff
@@ -13,114 +13,197 @@
-                            parameterList=parameterList.replaceAll("parameter",method.getParameterTypes()[0].getName()+" "+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value());
-                            stringBuilderParameter.append(parameterList);
-                            j++;
-                            }
-                        integrate = integrate.replaceAll("datatype", parameterList);
-                        integrate = integrate.replaceAll("integrationspecific", "return new ResponseEntity<String>(" + ((String) iterator.get(integrationName)).toLowerCase() + "Proxy." + method.getName() +"("+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value()+"), HttpStatus.OK);");
-
-                        proxy=proxy.replaceAll("path","\""+((RequestMapping)classNameInterfaces[0].getMethods()[i].getAnnotations()[2]).value()[0]+"\"");
-                        i++;
-                        proxy=proxy.replaceAll("parameter","@RequestParam(\""+((RequestParam)method.getParameters()[0].getAnnotations()[2]).value()+"\")"+method.getParameterTypes()[0].getName()+" "+((RequestParam) method.getParameters()[0].getAnnotations()[2]).value());
-                    } else {
-                        integrate = integrate.replaceAll("datatype", "");
-                        integrate = integrate.replaceAll("integrationspecific", "return new ResponseEntity<String>(" + ((String) iterator.get(integrationName)).toLowerCase() + "Proxy." + method.getName() + "(), HttpStatus.OK);");
-
-                        proxy=proxy.replaceAll("path","\""+((RequestMapping)classNameInterfaces[0].getMethods()[i].getAnnotations()[2]).value()[0]+"\"");
-                        i++;
-                        proxy=proxy.replaceAll("parameter","");
-                    }
-                    stringBuilderIntegrate.append(integrate);
-                    stringBuilderIntegrate.append("\n");
-
-
-                    stringBuilderProxy.append(proxy);
-                    stringBuilderProxy.append("\n");
-                } else {
-                    break;
-                }
+  public static void main(String[] args) throws Exception {
+
+    IntegrationName integrationName = new IntegrationName();
+    Field[] fieldList = integrationName.getClass().getFields();
+
+    for (Field iterator : fieldList) {
+
+      //Script to generate  CONTROLLER CODE(IntegrationNameAPI.java).
+      String content = IOUtils.toString(new FileReader(new File("TemplateName.java")));
+
+      FileWriter fileWriter =
+          new FileWriter(new File(iterator.get(integrationName) + "ApiController.java"));
+
+      String integration = IOUtils.toString(new FileReader(new File("IntegrationData.txt")));
+
+      String integrateString = IOUtils.toString(new FileReader(new File(
+          "/Users/prateekkoul/Downloads/spring-server-generated (1)/src/main/java/io/swagger/api/"
```

Comment:
can you use relative paths here? i'm skeptical this will work for anyone other than you right now.

Reply:
I replaced **absolute paths** to **relative paths** in recent commit on prateek.